RecoLocalCalo/HGCalRecProducers/python/HGCalUncalibRecHit_cfi.py

```diff
@@ -65,18 +66,22 @@
         tdcSaturation = hfnoseDigitizer.digiCfg.feCfg.tdcSaturation_fC,
         tdcOnset      = hfnoseDigitizer.digiCfg.feCfg.tdcOnset_fC,
         toaLSB_ns     = hfnoseDigitizer.digiCfg.feCfg.toaLSB_ns,
-        fCPerMIP      = cms.vdouble(1.25,2.57,3.88) #100um, 200um, 300um
+        fCPerMIP      = fCPerMIP_mpv
         ),
 
     algo = cms.string("HGCalUncalibRecHitWorkerWeights")
 )
 
 from Configuration.Eras.Modifier_phase2_hgcalV10_cff import phase2_hgcalV10
-phase2_hgcalV10.toModify( HGCalUncalibRecHit.HGCEEConfig , fCPerMIP = fCPerMIP_v10 ) 
-phase2_hgcalV10.toModify( HGCalUncalibRecHit.HGCHEFConfig , fCPerMIP = fCPerMIP_v10 )
+phase2_hgcalV10.toModify( HGCalUncalibRecHit.HGCEEConfig , fCPerMIP = fCPerMIP_mean ) 
+phase2_hgcalV10.toModify( HGCalUncalibRecHit.HGCHEFConfig , fCPerMIP = fCPerMIP_mean )
+
+from Configuration.Eras.Modifier_phase2_hgcalV16_cff import phase2_hgcalV16
+phase2_hgcalV16.toModify( HGCalUncalibRecHit.HGCEEConfig , fCPerMIP = fCPerMIP_mean ) 
+phase2_hgcalV16.toModify( HGCalUncalibRecHit.HGCHEFConfig , fCPerMIP = fCPerMIP_mean )
```

Comment:
Are these two configs the same as `phase2_hgcalV10`? If they are the same, I believe we don't need them because Era of Phase2C17I13M9 is based on C12 ==> C11 (but excludes HGCal V12) ==> C9 (still includes phase2_hgcalV10 and phase2_hgcalV11)

Reply:
Yes, they are the same. So, you say that Phase2C17I13M9 includes `phase2_hgcalV10` and will pick up the relevant values of the `phase2_hgcalV10` if `phase2_hgcalV16` is not present? Ok, then yes I will remove it now.

Reply:
Right.

Reply:
Ciao @srimanob this is a part that I am aware of and that I do not particularly like about eras and their chaining.
I did not suggest that change so that the choice would be *explicit* and to ease the process of removing old eras from the release in the future.
If we now want to obsolete `phase2_hgcalV10` we will need additional code to handle that.
I understand the usefulness of chaining and combining eras together, but I do not see it working in a situation like the one for `HGCAL`, in which we need to support, in the end, one detector and one configuration.
I'd therefore be in favour of reverting the last commit from Andreas.

Reply:
Hi @rovere 
I have no issue if you want to keep. I see that `phase2_hgcalV10` is used in several places so I understand that it is the basic customization you will have. So changing when clean-up is OK. But if you would like to keep it for this part first, I am also fine too.

Reply:
Ciao @srimanob yes, `V10` is used as a baseline pretty much everywhere. After some years of development, I'd have not coupled eras at all. In the end, reverting the change back will add just a couple of CPU cycles redoing what `V10` already did under the hood.
I'm still in favour of having the explicit call to `V16`.

The cleaning will not be trivial, I realize. As soon as the situation from the geometry point of view is more stable, we will address that, too.

Reply:
OK great, let me revert the last commit and push it.